CONTRIBUTING.md

```diff
@@ -8,20 +8,29 @@ To contribute icons, you only need an icon editor, a file explorer, a text edito
 
 For additional information on designing icons and samples, see [the Lawnicons Figma repository](https://www.figma.com/community/file/1227718471680779613) made by [Grabstertv](https://github.com/Grabstertv).
 
-### Artboard & Grid
+Need help? [Join Lawnchair on Discord](https://discord.com/invite/3x8qNWxgGZ).
+
+### Canvas & Sizes
 
 ![](docs/images/creating-icons-1-artboard.png)
 
-Each icon must fit the `160x160px` or `144x144px` (depending on the shape) content area size. It must not be smaller nor bigger than the specified sizes.
+#### Canvas
+The canvas size should be `192×192px` so that there is a safe zone around the icons to control consistency.
+#### Content area for all but square
+All but square icons must fit the `160×160px` content area size. Be careful with abstract icons: the long side should be `160px`, but the other side can be smaller. Don't forget to adjust the size as you change the stroke thickness.
+#### Content area for squares
+Square icons must fit the `154×154px` content area size. Squares should be smaller than other icons to maintain visual balance. Note that icons that mostly fit in a square are considered square. See the samples in [GitHub](docs/images/creating-icons-6-sample-icons.png) or [Figma](https://www.figma.com/file/YeHvAvz2g4vqqXGqgGLqRI/%F0%9F%AA%91-Lawnicons-Guidelines?type=design&node-id=307%3A282&mode=design&t=Bf94B5qZCVr9gV0b-1).
```

Comment:
> Note that icons that mostly fit in a square are considered square.

Just want to clarify, does this mean the icon needs a 1:1 ratio or will it be considered a square if it's 154x152. If 154x152 is acceptable then what is the limit till it's not considered a square anymore?

Reply:
All square icons should be 154×154px.

If the icon is 154×152px, I would ask to add 2px in that case. If the icon is, say, 154×150px, then you should probably look at the original — if it's not square, then you should change the size to 160×156px, for example.